sam-karis/airtech
# #ft register

## What does this PR do?
------
It adds register functionality

## Description of Task to be completed?
-----
- add user model
- add an index and register views, validators, and serializers
- add tests
- update settings


## How should this be manually tested?
------
clone the repo
check out to `ft-register-167660069`
add environment variables as defined in `.env_example`
Create a database and run `python manage.py migrate` to create relevant tables.


## Any background context you want to provide?
------
None

## What are the relevant pivotal tracker stories?
------
[register user](https://www.pivotaltracker.com/story/show/167660069)


## Screenshots (if appropriate)
------
#### Register successfully
![image](https://user-images.githubusercontent.com/22454909/62813275-8ebbf980-bb12-11e9-83c2-58f4a005f034.png)

#### Register with invalid email
![image](https://user-images.githubusercontent.com/22454909/62813315-c7f46980-bb12-11e9-9077-39bed2f7efdf.png)

#### Register with unmatching passwords
![image](https://user-images.githubusercontent.com/22454909/62813420-5cf76280-bb13-11e9-859f-f880d1082245.png)
